docs/Introduction/tutorials/advanced-tutorial.md

```diff
@@ -11,47 +11,64 @@ metadata:
+# 3. What are Adapters?
 
-## 1c. Core Adapters
+Each oracle job has a configured set of tasks it must complete when it is run. These tasks are defined by the [**Adapters**](../core-adapters/) they support. Adapters are split into two subcategories:
 
-Each oracle job has a configured set of tasks it needs to carry out when it is run. These tasks are defined by what [Adapters](../core-adapters/) they support. For example: if a job needs to make a GET request to an API, find a specific unsigned integer field in a JSON response, then submit that back to the requesting contract, it would need a job with the following Core Adapters:
-- [HttpGet](../core-adapters/#httpget) - Call the API
-- [JsonParse](../core-adapters/#jsonparse) - Parse the JSON and retrieve the desired data
-- [EthUint256](../core-adapters/#ethuint256) - Convert the data to Ethereum compatible data type (uint256)
-- [EthTx](../core-adapters/#ethtx)  - Submit the transaction to the chain, completing the cycle.
+- **Core Adapters** - These are adapters that come built-in to each node. (examples: HttpGet, EthUint256, etc)
+- **External Adapters** - These are custom adapters built by node operators and community members, which perform specific tasks like calling a particular endpoint with a specific set of parameters (like authentication secrets that shouldn't be publicly visible smart contracts).
 
-Let's walk through a real example, where we retrieve 24 volume of the <a href="https://min-api.cryptocompare.com/data/pricemultifull?fsyms=ETH&tsyms=USD" target="_blank">ETH/USD pair</a> from the cryptocompare API.
+## Core Adapters
 
-### Core Adapters Example
+If a job needs to make a GET request to an API, find a specific unsigned integer field in a JSON response, then submit that back to the requesting contract, it would need a job containing the following Core Adapters:
+- [HttpGet](../core-adapters/#httpget) calls the API
+- [JsonParse](../core-adapters/#jsonparse) parses the JSON and retrieve the desired data
+- [EthUint256](../core-adapters/#ethuint256) converts the data to Ethereum compatible data type (uint256)
+- [EthTx](../core-adapters/#ethtx) submits the transaction to the chain, completing the cycle.
 
-1. [HttpGet](../core-adapters/#httpget) - Calls the API and returns the body of an HTTP GET result for [ETH/USD pair](https://min-api.cryptocompare.com/data/pricemultifull?fsyms=ETH&tsyms=USD).  Example:
+Let's walk through a real example, where you will retrieve 24 volume of the [ETH/USD pair](https://min-api.cryptocompare.com/data/pricemultifull?fsyms=ETH&tsyms=USD) from the cryptocompare API.
+
+1. [HttpGet](../core-adapters/#httpget) calls the API and returns the body of an HTTP GET result for [ETH/USD pair](https://min-api.cryptocompare.com/data/pricemultifull?fsyms=ETH&tsyms=USD).  Example:
 ```json
```

Comment:
could we use the `TOML` version of this? 

It looks like:

```toml
type = "directrequest"
schemaVersion = 1
name = "Get > Uint256"
contractAddress = "ORACLE_CONTRACT_ADDRESS"
maxTaskDuration = "0s"
observationSource = """
    decode_log   [type=ethabidecodelog
                  abi="OracleRequest(bytes32 indexed specId, address requester, bytes32 requestId, uint256 payment, address callbackAddr, bytes4 callbackFunctionId, uint256 cancelExpiration, uint256 dataVersion, bytes data)"
                  data="$(jobRun.logData)"
                  topics="$(jobRun.logTopics)"]

    decode_cbor  [type=cborparse data="$(decode_log.data)"]
    fetch        [type=http method=GET url="$(decode_cbor.get)"]
    parse        [type=jsonparse path="$(decode_cbor.path)" data="$(fetch)"]
    multiply     [type=multiply input="$(parse)" times=100]     
    encode_data  [type=ethabiencode abi="(uint256 value)" data=<{ "value": $(multiply) }>]
    encode_tx    [type=ethabiencode
                  abi="fulfillOracleRequest(bytes32 requestId, uint256 payment, address callbackAddress, bytes4 callbackFunctionId, uint256 expiration, bytes32 data)"
                  data=<{"requestId": $(decode_log.requestId), "payment": $(decode_log.payment), "callbackAddress": $(decode_log.callbackAddr), "callbackFunctionId": $(decode_log.callbackFunctionId), "expiration": $(decode_log.cancelExpiration), "data": $(encode_data)}>
                 ]
    submit_tx    [type=ethtx to="ORACLE_CONTRACT_ADDRESS" data="$(encode_tx)"]

    decode_log -> decode_cbor -> fetch -> parse -> multiply -> encode_data -> encode_tx -> submit_tx
"""
externalJobID = "19747015-6833-40e7-b3fe-0fa1e76b843b"
```

Reply:
Perhaps I'm scope creeping requesting this... Maybe it's a bigger project to update all JSON job samples to TOML. 

let me know what you think.

Reply:
Captured here: https://github.com/smartcontractkit/documentation/issues/500

We'll address this in a followup.